test/integration/multipleInheritance.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { MultipleInheritanceClient } from "./generated/multipleInheritance/src/multipleInheritanceClient";
+import { assert } from "chai";
+import { MultipleInheritanceClientGetCatResponse } from "./generated/multipleInheritance/src/models";
+
+describe("MultipleInheritance", () => {
+  let client: MultipleInheritanceClient;
+
+  beforeEach(() => {
+    client = new MultipleInheritanceClient();
+  });
+
+  it("should get a pet", async () => {
+    const { _response, ...result } = await client.getPet();
```

Comment:
Ah, that's a good trick. I hadn't thought of doing that when I needed to write similar tests and went with a more complicated approach.

Reply:
It just occurred to me today, I was also doing other complicated stuff. I think I'll stick with this in future tests